<ui version="4.0" >
 <class>PhotoDialog</class>
 <widget class="QWidget" name="PhotoDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="verticalSpacing" >
    <number>0</number>
   </property>
   <item row="0" column="0" >
    <widget class="QLabel" name="photoiconlabel" >
     <property name="text" >
      <string/>
     </property>
     <property name="pixmap" >
      <pixmap resource="images.qrc" >:/images/lphoto16.png</pixmap>
     </property>
     <property name="scaledContents" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QLabel" name="label_7" >
     <property name="text" >
      <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:8pt; font-weight:600;">Photo View&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>391</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="3" >
    <widget class="QSplitter" name="splitter_2" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="layoutWidget" >
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QSplitter" name="splitter" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QTreeWidget" name="peerTreeWidget" >
          <property name="contextMenuPolicy" >
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <column>
           <property name="text" >
            <string>Peer</string>
           </property>
          </column>
          <column>
           <property name="text" >
            <string>Slideshow</string>
           </property>
          </column>
          <column>
           <property name="text" >
            <string>Photo</string>
           </property>
          </column>
         </widget>
         <widget class="QTreeWidget" name="photoTreeWidget" >
          <property name="contextMenuPolicy" >
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <column>
           <property name="text" >
            <string>Thumb Image</string>
           </property>
          </column>
          <column>
           <property name="text" >
            <string>Image Name</string>
           </property>
          </column>
          <column>
           <property name="text" >
            <string>Comment</string>
           </property>
          </column>
          <column>
           <property name="text" >
            <string>Date</string>
           </property>
          </column>
          <column>
           <property name="text" >
            <string>Location</string>
           </property>
          </column>
          <column>
           <property name="text" >
            <string>Size</string>
           </property>
          </column>
          <column>
           <property name="text" >
            <string>PeerId</string>
           </property>
          </column>
          <column>
           <property name="text" >
            <string>PhotoId</string>
           </property>
          </column>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QPushButton" name="expandButton" >
           <property name="text" >
            <string/>
           </property>
           <property name="icon" >
            <iconset resource="images.qrc" >:/images/add_24x24.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="addButton" >
           <property name="text" >
            <string>Add Photo(s)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="addShowButton" >
           <property name="text" >
            <string>Add Photo SlideShow</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="updateButton" >
           <property name="text" >
            <string>Update Details</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget" >
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Photo </string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QLineEdit" name="photoNameEdit" />
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="label_3" >
         <property name="text" >
          <string>Date</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QLineEdit" name="photoDateEdit" />
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="label_4" >
         <property name="text" >
          <string>Location</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="QLineEdit" name="photoLocEdit" />
       </item>
       <item row="3" column="0" >
        <widget class="QLabel" name="label_5" >
         <property name="text" >
          <string>Description</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" >
        <widget class="QLineEdit" name="photoDescEdit" />
       </item>
       <item row="4" column="0" >
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>Comment</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" >
        <widget class="QTextEdit" name="photoCommentEdit" />
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images.qrc" />
 </resources>
 <connections/>
</ui>
